python/paddle/fluid/tests/unittests/test_activation_op.py

```diff
@@ -19,6 +19,24 @@
 import paddle.fluid.core as core
 from op_test import OpTest
 from scipy.special import expit, erf
+import paddle.fluid as fluid
+from paddle.fluid import compiler, Program, program_guard
+
+
+class TestSqrtOpError(OpTest):
+    def test_errors(self):
+        with program_guard(Program(), Program()):
+            # The input type of sign_op must be Variable or numpy.ndarray.
+            in1 = 1
+            self.assertRaises(TypeError, fluid.layers.sqrt, in1)
+            # The input dtype of sign_op must be float32, float64.
```

Comment:
sign_op-》sqrt_op, typo
must be float16, float32, float64.

Reply:
Done